releasenotes/notes/botocore_dsm_sqs-02df9afa4b4bb503.yaml

```
@@ -0,0 +1,3 @@
+---
+features:
+botocore: Added message attributes to sqs spans to support data streams monitoring.
```

builddocs is failing due to formatting issues:
```suggestion
    botocore: Added message attributes to sqs spans to support data streams monitoring.
```